Userland/Libraries/LibWeb/HTML/Window.h

```diff
@@ -288,6 +291,9 @@ class Window final
     // https://html.spec.whatwg.org/multipage/interaction.html#user-activation-data-model
     HighResolutionTime::DOMHighResTimeStamp m_last_activation_timestamp { NumericLimits<double>::max() };
 
+    // https://html.spec.whatwg.org/multipage/interaction.html#last-history-action-activation-timestamp
+    HighResolutionTime::DOMHighResTimeStamp m_last_history_action_activation_timestamp { NumericLimits<double>::max() };
```

Comment:
The spec says this should be initialized to positive infinity:
```suggestion
    HighResolutionTime::DOMHighResTimeStamp m_last_history_action_activation_timestamp { Infinity<double> };
```
`NumericLimits<double>::max()` returns the highest finite number representable by `double`

Reply:
Thank you. I hadn't realised `Infinity` existed - and just did the same thing as the code above (for last activation timestamp).

I've corrected the code I've added, and the code above (https://github.com/SerenityOS/serenity/pull/24437/commits/d1cdd5a462ff06af551b1bf1a5d398873a45f331) :)